jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/server/Handshaker.java

```diff
@@ -23,19 +23,16 @@
 import javax.servlet.http.HttpServletResponse;
 
 import org.eclipse.jetty.websocket.core.FrameHandler;
+import org.eclipse.jetty.websocket.core.server.internal.HandshakerSelector;
 import org.eclipse.jetty.websocket.core.server.internal.RFC6455Handshaker;
+import org.eclipse.jetty.websocket.core.server.internal.RFC8441Handshaker;
 
 public interface Handshaker
 {
     static Handshaker newInstance()
     {
-        return new RFC6455Handshaker();
+        return new HandshakerSelector(new RFC6455Handshaker(), new RFC8441Handshaker());
```

Comment:
Can we check if HTTP2 is on the classpath of something and if not only return a RFC6455Handshaker?